src/cloudflare/workers.ts

```diff
@@ -11,3 +11,8 @@ export const WorkerEntrypoint = entrypoints.WorkerEntrypoint;
 export const DurableObject = entrypoints.DurableObject;
 export const RpcStub = entrypoints.RpcStub;
 export const RpcTarget = entrypoints.RpcTarget;
+
+/* eslint-disable */
+import { default as flags } from 'workerd:compatibility-flags';
+export const compatibilityFlags = (flags as any).compatibilityFlags;
```

Comment:
I think we can remove eslint-disable here. Doesn't this work? 

```
export {
  compatibilityFlags: flags.compatibilityFlags,
}
```

Reply:
No, this doesn't work and the eslint-disable complains